lexibank_bds.py

```diff
@@ -0,0 +1,138 @@
+import pathlib
+from collections import defaultdict
+
+import attr
+from lingpy import Wordlist
+from collabutils.edictor import fetch
+from tabulate import tabulate
+from pylexibank import Dataset as BaseDataset, Language, Lexeme
+from clldutils.misc import slug
+
+
+@attr.s
+class Word(Lexeme):
+    autoborid = attr.ib(
+        default=None,
+        converter=lambda s: None if s in ('0', 0) else int(s),
+        metadata={'dc:description': 'Automatically inferred borrowing Identifier'}
```

Comment:
Here we could add documentation about the automatically inferred columns in the wordlist.

Reply:
How cool, I did not know that I can add the description via the metadata! I should do that in other columns as well then.